sdk/appcenter-analytics/src/main/java/com/microsoft/appcenter/analytics/Analytics.java

```diff
@@ -875,6 +911,26 @@ private void setDefaultTransmissionTarget(String transmissionTargetToken) {
         }
     }
 
+    /**
+     * Set trigger interval. The interval should be between 3 seconds and 1 day.
+     * Should be called before the service is started.
+     *
+     * @param seconds Set latency of sending events to Analytics.
+     * @return succeed <code>true</code> if set succeed, <code>false</code> otherwise.
+     */
+    private boolean setInstanceTransmissionInterval(int seconds) {
+        if (mChannel != null) {
+            AppCenterLog.error(LOG_TAG, "Transmission interval should be set before the service is started.");
+            return false;
+        }
+        if (seconds < TRANSMISSION_INTERVAL_MINIMUM || seconds > TRANSMISSION_INTERVAL_MAXIMUM) {
+            AppCenterLog.error(LOG_TAG, "The transmission interval is invalid. The value should be between 3 seconds and 1 day");
```

Comment:
To be future proof in case we change the min/max range but forget to modify the hardcoded value in this error message, let's use `TRANSMISSION_INTERVAL_MINIMUM` and `TRANSMISSION_INTERVAL_MAXIMUM` to format the string.